src/Form/ReactionRuleEditForm.php

```diff
@@ -95,4 +135,29 @@ protected function getRuleConfig() {
     return $this->entity;
   }
 
+  /**
+   * Checks if the event of the current rule is registered into the container.
+   *
+   * @return true | false
```

Comment:
See https://www.drupal.org/coding-standards/docs#return on how to document return values.